src/app/data-explorer/selection-compact/selection-compact.component.html

```diff
@@ -0,0 +1,49 @@
+<!--
+Copyright 2022-2023 University of Oxford
+and Health and Social Care Information Centre, also known as NHS Digital
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+-->
+<div
+  class="highlight-box mdm-selection-compact"
+  *ngIf="(selectedElements$ | async)?.length != 0"
```

Comment:
In this case I think it's correct because we may also have _undefined_ - we want the condition to be true if X != 0 AND X != undefined. It would be clearer to write the conditions separately but the cost would be to repeat the binding twice. I'm tempted to leave it but I don't like it much

Reply:
OK, I didn't think of an `undefined` check. Acceptable to leave as-is then.